src/qlever/commands/ui.py

```diff
@@ -59,57 +102,133 @@ def execute(self, args) -> bool:
                 log.info("")
 
         # Construct commands and show them.
-        server_url = f"http://{args.host_name}:{args.port}"
+        pull_latest_image = "/" in args.ui_image and not args.no_pull_latest
```

Comment:
note: Locally built images can be tagged with with names that suggest that they stem from a repository, they can be even be tagged as `docker.io/adfreiburg/qlever-ui:latest`.
suggestion: Document the assumption that local images don't contain slashes.

Reply:
Yeah, that was really just a quick hack and it was useful for debugging and testing this. The question is whether a normal user will ever need this? If it is rare, we can just have a `--no-pull-latest` option or something like that. Which, in fact, I had, before I hacked it like this.